### ISSUE TITLE
Difference between DevicesUpdateCallbackInterval and ScanMode

### ISSUE THREAD

**Issue by robachmann** [kontaktio/kontakt-android-sdk#95]
Could you please explain how these two settings work together? 

```
IBeaconScanContext iBeaconScanContext = new IBeaconScanContext.Builder()
                .setDevicesUpdateCallbackInterval(TimeUnit.SECONDS.toMillis(1))
```

and

```
ScanContext scanContext = new ScanContext.Builder()
                .setIBeaconScanContext(iBeaconScanContext)
                .setScanPeriod(ScanPeriod.RANGING)
                .setScanMode(ProximityManager.SCAN_MODE_LOW_LATENCY)
```

Which values should I set to get as many measures as possible (e.g. with an `interval=200ms` configured in the Konakt.io panel for a beacon I would expect to receive 5 notifications per second from the Kontakt.io SDK).

**Comment by michaldrabik** [comment]
@240ch The device update callback interval prevents the event with DEVICES_UPDATE type from being called instantly.

**Comment by robachmann** [comment]
@michaldrabik interesting, thank you. So it wouldn't make sense to set the `DevicesUpdateCallbackInterval` too high if I already stress the device in LOW_LATENCY mode, right? ;-) 

I tested a setup where I set TX power mode to 7 and interval to 100ms, then I measured how many packets I receive with kontakt.io sdk within one minute. I was surprised that I only captured 200 packets instead of 600 (60 seconds \* 10 per second (=100ms interval)). Is there something you would recommend to ensure capturing as many packets as possible (regardless of power consumption)?
